src/test/java/com/bunq/sdk/http/BunqHeaderTest.java

```diff
@@ -0,0 +1,49 @@
+package com.bunq.sdk.http;
+
+import org.junit.Test;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.junit.Assert.*;
+
+public class BunqHeaderTest {
+    @Test
+    public void parse() {
+        // parse works case-insensitive
+        assertEquals(BunqHeader.clientResponseId,BunqHeader.parse("X-Bunq-Client-Response-Id").get());
+        assertEquals(BunqHeader.clientResponseId,BunqHeader.parse("x-bunq-client-response-id").get());
+    }
+
+    @Test
+    public void isBunq() {
+        List<BunqHeader> nonBunqHeaders = Arrays.asList(BunqHeader.cacheControl,BunqHeader.contentType,BunqHeader.userAgent);
+
+        assertEquals(nonBunqHeaders, Stream.of(BunqHeader.values()).filter(h->!h.isBunq()).collect(Collectors.toList()));
+    }
+
+    @Test
+    public void getOrDefault() {
+        BunqHeader h = BunqHeader.clientResponseId;
+
+        assertEquals("test-id",h.getOrDefault(Collections.singletonMap("x-bunq-client-response-id","test-id")));
+        assertEquals("Could not determine response id.",h.getOrDefault(Collections.singletonMap("x-bunq-some-other-header","test-id")));
+        assertEquals("Could not determine response id.",h.getOrDefault(Collections.emptyMap()));
+    }
+
+    @Test
+    public void addToMap() {
+        Map<String,String> headers = new HashMap<>();
+
+        //sut
+        BunqHeader.language.addTo(headers,null);
+        BunqHeader.geolocation.addTo(headers,null);
+        BunqHeader.userAgent.addTo(headers,"test-agent");
+
+        // verify
+        assertEquals("en_US",headers.get("X-Bunq-Language"));
+        assertEquals("0 0 0 0 000",headers.get("X-Bunq-Geolocation"));
+        assertEquals("test-agent",headers.get("User-Agent"));
+    }
+}
```

Comment:
missing new line at EOF

Reply:
i changed this in BunqHeaderTest, see https://github.com/bunq/sdk_java/pull/102/commits/ad67adaf991efcb3a5135278c0f3108572c4783b